mne/surface.py

```diff
@@ -606,7 +606,7 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_curvature(filepath):
+def read_curvature(filepath, binary=True):
     """Load in curavature values from the ?h.curv file."""
```

Comment:
I am surprised there is no docstring in this funciton and CIs don't complain. Can you add it?

Reply:
It's not checked because `mne.surface` is not in the `public_modules` list in `mne/tests/test_docstring_parameters.py` list. @GuillaumeFavelier feel free to add it. You might then need to fix some docstrings and/or add entries to `docstring_ignores`